apps/judicial-system/web/src/utils/formHelper.ts

```diff
@@ -0,0 +1,190 @@
+import { Case, UpdateCase } from '@island.is/judicial-system/types'
+import {
+  padTimeWithZero,
+  parseString,
+  parseTime,
+  replaceTabs,
+} from './formatters'
+import { validate, Validation } from './validate'
+import formatISO from 'date-fns/formatISO'
+
+export const RemoveTabsValidateAndSet = (
```

Comment:
Why an upper case name here and not else where?

Reply:
fixed